src/com/amazon/ion/benchmark/WriteRandomIonValues.java

```diff
@@ -76,10 +76,8 @@ class WriteRandomIonValues {
     final static private Set<String> VALID_STRING_SYMBOL_CONSTRAINTS = Collections.unmodifiableSet(new HashSet<>(Arrays.asList("regex", "codepoint_length")));
     final static private Set<String> VALID_DECIMAL_CONSTRAINTS = Collections.unmodifiableSet(new HashSet<>(Arrays.asList("precision", "scale")));
     final static private Set<String> VALID_TIMESTAMP_CONSTRAINTS = Collections.unmodifiableSet(new HashSet<>(Arrays.asList("timestamp_offset", "timestamp_precision")));
-    // 0001-01-01T00:00:00.0Z in millis.
-    final static private BigDecimal MINIMUM_TIMESTAMP_IN_MILLIS_DECIMAL = new BigDecimal(-62135769600000L);
-    // 10000T in millis, upper bound exclusive.
-    final static private BigDecimal MAXIMUM_TIMESTAMP_IN_MILLIS_DECIMAL = new BigDecimal(253402300800000L);
+    // Create a range which contains the default lower bound and upper bound values.
+    final static private Range RANGE = new Range(SYSTEM.newList( SYSTEM.newDecimal(62135769600000L), SYSTEM.newDecimal(253402300800000L)));
```

Comment:
Should this be `DEFAULT_RANGE`?

Reply:
> Should this be `DEFAULT_RANGE`?

Yes, the constant name should be replaced. Cause `DEFAULT_RANGE` has been used before, in the next commit it will be replaced with `DEFAULT_TIMESTAMP_IN_MILLIS_DECIMAL_RANGE`.